opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/authenticate/embedded/Xs2aAisReportSelectedScaMethod.java

```diff
@@ -46,6 +48,9 @@ protected void doRealExecution(DelegateExecution execution, Xs2aContext context)
                 params.getBody()
         );
 
+        byte[] imageData = consentRepository.findByPsuId(context.getPsuId()).getImageData();
```

Comment:
You should place the image data in Xs2aContext - it will persisted and encrypted opaquely.
Line
```
ContextUtil.getAndUpdateContext(
```
updates it, so that it can be used later

Reply:
Now it is stored in the Xs2aContext